build.sbt

```diff
@@ -20,6 +20,16 @@ def projectName(project: sbt.ResolvedProject): String = {
   convertCamelKebab(project.id)
 }
 
+lazy val startupTransition: State => State = { s: State =>
+  if (System.getProperty("METALS_ENABLED") != null) "^^1.2.6" :: s
```

Comment:
`METALS_ENABLED=false` :trollface: 

We only set it to `true` so no worries :p

Reply:
@olafurpg I should have asked for your review since you gave me the code. :-)